DependencyInjection/SwarrotExtension.php

```diff
@@ -95,6 +95,7 @@ public function buildCommand(ContainerBuilder $container, string $name, array $c
             ->addTag('console.command', [
                 'command' => 'swarrot:consume:'.$name,
             ])
+            ->setPublic(true)
```

Comment:
I had an error on unit test Tests/DependencyInjection/SwarrotExtensionTest.php

I don't know if I should remove assert or set service public. Let me know.

Reply:
As the service as public before, let's keep it like this. :ok_hand: